demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -108,30 +108,13 @@ public void onClick(View v) {
             }
         });
 
-        final TextView downloadedTitles = findViewById(R.id.textview_completed_download_title);
-        View buttonShowCompletedDownloads = findViewById(R.id.button_show_completed_downloads);
-        buttonShowCompletedDownloads.setOnClickListener(new View.OnClickListener() {
+        View buttonLogFilesDir = findViewById(R.id.btn_log_files_dir);
+        buttonLogFilesDir.setOnClickListener(new View.OnClickListener() {
             @Override
-            public void onClick(View view) {
```

Comment:
This didn't actually do anything when it was clicked. Replaced it with a button that dumps the file names in the local storage directory to Logcat.